pkg/command/context.go

```diff
@@ -250,19 +253,59 @@ func createCtx(_ *cobra.Command, args []string) (err error) {
 	}
 
 	// Sync all required plugins
-	_ = syncContextPlugins(ctx.ContextType)
+	_ = syncContextPlugins(cmd, ctx.ContextType, false)
```

Comment:
Thanks to the nice way you wrote the code, I think you only need to change this `false` to `true` to print the list of plugins to be installed for `tanzu context create`

Reply:
updated.